roadmap/implementors-guide/src/type-definitions.md

```diff
@@ -173,6 +173,34 @@ enum MisbehaviorArbitrationMessage {
 }
 ```
 
+## Provisioner Message
+
+```rust
+/// This data becomes intrinsics or extrinsics which should be included in a future relay chain block.
+enum ProvisionableData {
+  /// This bitfield indicates the availability of various candidate blocks.
+  Bitfield(Hash, SignedAvailabilityBitfield),
+  /// The Candidate Backing subsystem believes that this candidate is backable, pending availability.
+  /// TODO: do we need any more data than this?
+  BackableCandidate(Hash, CandidateReceipt),
```

Comment:
Should be the `BackedCandidate` struct, which contains the candidate and its backing. That's what's actually submitted on-chain to the `InclusionInherent` module